### PR TITLE
[doc] LR scheduler example fix

### PR DESCRIPTION
**Pull request by daspartho**
Fixes issue #86208 
As suggested in the issue, updated the LR scheduler example to use a regular nn.Module like the other examples on the same page.

### CODE REVIEW

**Review by soulitzer — COMMENTED** [review]
Thanks for the update! Looks OK to me, do we want to define module to be something like Linear though?

**Comment by daspartho** [comment]
That could be added; I tried to make the example look like the other examples on the same page. (https://pytorch.org/docs/stable/optim.html#how-to-use-an-optimizer)

**Review by soulitzer — APPROVED** [review]
Oh that sounds OK to me as well
